backend/handler/db_handler.py

```diff
@@ -75,56 +69,51 @@ def add_rom(self, rom: Rom) -> None:
         except ProgrammingError as e:
             self.raise_error(e)
 
-
     def get_roms(self, p_slug: str) -> list[Rom]:
         try:
-            with self.session.begin() as s:
-                return s.scalars(select(Rom).filter_by(p_slug=p_slug).order_by(Rom.file_name.asc())).all()
+            return select(Rom).filter_by(p_slug=p_slug).order_by(Rom.file_name.asc())
```

Comment:
Only change here is removing the scalar

Reply:
Yes totally agree. To be honest I was learning SQAlchemy while developing RomM, so I see now a lot of imporvements with it.